docs/aws.md

```diff
@@ -44,16 +44,44 @@ brew install kubernetes-cli
 
 ## Setup your environment
 
-### Setting up a kops IAM user
+## Create an IAM user for kops
```

Comment:
Can we move this block below `Testing your DNS setup` and above `Setting up a state store for your cluster?` The way it reads we are asking users to use the `aws` cli before we tell them how to install it :)